content/platform/how-tos/minimise-costs.md

```diff
@@ -0,0 +1,35 @@
++++
+title = "Enable Preemptible Instances"
+weight = 4
+chapter = false
+pre = ""
++++
+
+{{% notice warning %}}
+Preemptible instances are not recommended for running workloads that are not fault tolerant. Instances will only run for 24 hours and will be turned off afterwards.
+{{% /notice %}}
+
+{{% notice note %}}
+Preemption works well if you need to cut down costs for your clusters. See [minimising costs](../../minimise-costs).
+{{% /notice %}}
+
+
+## Cluster setup with minimal costs
+Create a cluster with autoscaling disabled, preemptible nodes, HDD disk and `e2-medium` disk type.
```

Comment:
is disabling autoscaling important here? some readers may think that it is a requirement of preemtible nodes

Reply:
by `autoscaling` here you mean "Node Auto Provisioning" or Pool Size autoscaling?